getting_started/devpreview_faq.adoc

```diff
@@ -56,20 +56,23 @@ provision a fresh set of resources for you as soon as they become available.
 WHAT LANGUAGES ARE SUPPORTED?::
 The {product-title} (Next Gen) Developer Preview currently supports:
 
-- Node.js (0.10)
+- Jenkins (1.6)
+- .NET Core (1.0)
+- Node.js (0.10, 4)
 - PHP (5.5, 5.6)
-- Python (2.7, 3.3, 3.4)
-- Ruby (2.0, 2.2)
+- Python (2.7, 3.3, 3.4, 3.5)
+- Ruby (2.0, 2.2, 2.3)
 - Perl (5.16, 5.20)
 - Java (6, 7, 8, EE) is available via optional JBoss Middleware Services (JBoss
 EAP and JBoss Web Server)
 
 WHAT DATABASE SERVICES ARE SUPPORTED?::
 The {product-title} (Next Gen) Developer Preview currently supports:
 
-- MongoDB (2.4, 2.6)
+- MariaDB (10.1)
+- MongoDB (2.4, 2.6, 3.2)
 - MySQL (5.5, 5.6)
-- PostgreSQL (9.2, 9.4)
+- PostgreSQL (9.2, 9.4, 9.5)
 
 WHAT JBOSS MIDDLEWARE SERVICES ARE AVAILABLE IN THE DEVELOPER PREVIEW?::
 JBoss EAP and JBoss Web Server are available to try during the {product-title}
```

Comment:
I would consider modifying this FAQ answer to simply link to https://www.openshift.com/features/cartridges.html